src/mesh_smoothing.rs

```diff
@@ -62,30 +81,41 @@ pub fn laplacian_smoothing_with_anchors(
 #[allow(dead_code)]
 pub fn laplacian_smoothing_with_anchors_stop_when_stable(
     geometry: &Geometry,
-    max_iterations: u8,
+    max_iterations: u32,
     fixed_vertex_indices: &[u32],
 ) -> Geometry {
-    laplacian_smoothing_with_anchors_full(geometry, max_iterations, fixed_vertex_indices, true)
+    let vertex_to_vertex_topology = mesh_topology_analysis::vertex_to_vertex_topology(geometry);
+    let (g, _, _) = laplacian_smoothing_with_anchors_full(
+        geometry,
+        vertex_to_vertex_topology,
+        max_iterations,
+        fixed_vertex_indices,
+        true,
+    );
+    g
 }
 
-fn laplacian_smoothing_with_anchors_full(
+/// returns (smooth_geometry: Geometry, executed_iterations: u32, stable: bool)
+pub fn laplacian_smoothing_with_anchors_full(
```

Comment:
I think there's now way too many of the `laplacian_smoothing_*` functions. Let's keep just the most generic one (and call it just `laplacian_smoothing`) and use that everywhere. Also make sure its docs are up to date and correctly formatted. Let's keep the tests for the various variants, just inline the function with concrete parameters set there.

Reply:
@janper Almost done, but you still forgot the rename to `laplacian_smoothing`